Source/Fuse.Share/ShareModule.uno

```diff
@@ -144,8 +144,8 @@ namespace Fuse.Share
 			{
 				var description = args.Length>2 ? "" + args[2] : "";
 				if defined(android)
-				{
-					AndroidShareImpl.ShareFile("file://" + path, type, description);
+				{	
```

Comment:
lol, my bad

Reply:
Sorted.

Reply:
Please squash the fix into the right commit :)

Reply:
I don't know what you mean; what is the "right commit", please explain.

Reply:
We try to avoid introducing a mistake in one commit and then fixing it in a later. Instead, you should rebase your branch so this white-space isn't inserted in the first place.

Reply:
Oh my gosh, sigh, okay...

Reply:
There you go, if I rebase to my original commit, I think this PR will close.